### PR TITLE
More contributing documentation fixes

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 We're so glad you're thinking about contributing to an 18F open source project! If you're unsure or afraid of anything, just ask or submit the issue or pull request anyways. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contribution, and don't want a wall of rules to get in the way of that.
 
+*Disclaimer:* Everthing in this guide is not set in stone and subject to change based on changing project requirements and tools.
```
Comment (jcscottiii) [inline CONTRIBUTING.md]: `Everthing` - > `Everything`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -200,7 +202,7 @@ Here are some basic rules to work with action creators successfully:
     return Promise.resolve(spaces);
   }
   ```
-- There should not be more then on action with the same data.
+- There should not be more then one action with the same data.
```
Comment (jcscottiii) [inline CONTRIBUTING.md]: `then` -> `than`

File: CONTRIBUTING.md (hunk 1 above)
Comment (rememberlenny) [inline CONTRIBUTING.md]: Made updates to merge

File: CONTRIBUTING.md (hunk 2 above)
Comment (rememberlenny) [inline CONTRIBUTING.md]: Made updates to merge

**Comment by rememberlenny** [comment]
@jcscottiii Made the updates. Good for merge.

**Pull request merged**